src/main/views/all-judgments.njk

```diff
@@ -41,7 +41,7 @@
               <td class="govuk-table__cell">
                 {{ govukTag({
                     text: app.displayStatus,
-                    classes: "govuk-tag" + app.statusColor
+                    classes: "no-text-transform govuk-tag" + app.statusColor
```

Comment:
We don't need to do this as part of our ticket, but it will solve one of the bugs QA complain about